recipes/rosco/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "rosco" %}
+{% set version = "2.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/nrel/rosco/archive/master.tar.gz
+  sha256: 06e2f01a8cc20d8fe271ad987f58e1996ab5352bd35e69f16a449c182860fcde
```

Comment:
This is not a released version

Reply:
@nikhar-abbas can you create a new release and update the recipe?

Reply:
Fixed!